core/src/components/fab-button/fab-button.scss

```diff
@@ -199,7 +199,7 @@
 // --------------------------------------------------
 
 :host(.fab-button-small) {
-  @include margin(($fab-size - $fab-small-size) * 0.5);
+  @include margin($fab-button-small-margin);
```

Comment:
This was always 8px, so I just made it its own variable

$fab-size: 56
$fab-small-size: 40

(56 - 40) * 0.5 = 16 * 0.5 = 8